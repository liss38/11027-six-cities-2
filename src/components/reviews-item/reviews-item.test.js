import React from 'react';
import renderer from 'react-test-renderer';
import ReviewsItem from './reviews-item.jsx';

it(`ReviewsItem rendered correctly`, () => {
  const tree = renderer
    .create(<ReviewsItem
      review={{
        name: `Lena`,
        avatarUrl: `img/avatar.svg`,
        rating: 4.2,
        date: `April 2019`,
        description: `Текст отзыва`,
      }}
    />)
    .toJSON();

  expect(tree).toMatchSnapshot();
});
