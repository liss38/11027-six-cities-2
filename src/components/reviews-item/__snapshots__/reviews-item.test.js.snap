// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ReviewsItem rendered correctly 1`] = `
<li
  className="reviews__item"
>
  <div
    className="reviews__user user"
  >
    <div
      className="reviews__avatar-wrapper user__avatar-wrapper"
    >
      <img
        alt="Lena"
        className="reviews__avatar user__avatar"
        height="54"
        src="img/avatar.svg"
        width="54"
      />
    </div>
    <span
      className="reviews__user-name"
    >
      Lena
    </span>
  </div>
  <div
    className="reviews__info"
  >
    <div
      className="reviews__rating rating"
    >
      <div
        className="reviews__stars rating__stars"
      >
        <span
          style={
            Object {
              "width": "84.00000000000001%",
            }
          }
        />
        <span
          className="visually-hidden"
        >
          Rating
        </span>
      </div>
    </div>
    <p
      className="reviews__text"
    >
      Текст отзыва
    </p>
    <time
      className="reviews__time"
      dateTime="2019-04-24"
    >
      April 2019
    </time>
  </div>
</li>
`;
